doc/dev_guide/release_management.rst

```diff
@@ -115,6 +115,14 @@ channel (`tarantool_news <https://t.me/tarantool_news>`_).
 We also publish nightly builds, and use the fourth slot in the version
 identifier to designate the nightly build number.
 
+.. important::
+
+   Only a version with ``0`` in the fourth slot, namely ``X.Y.Z-0-<hash>`` is
+   considered to be a *release*. Further nightly builds in the same branch
+   although published are **not** releases. For example, ``2.5.1-0-<hash>`` is
```

Comment:
> Further nightly builds in the same branch although published are **not** releases.

This is nitpicking, but I would try to reword it to make simpler. Like so, maybe:

> Packages that are published with non-zero fourth slot are nightly builds, not releases.

Reply:
re-writed